spec/concepts/stock/task/load_spec.rb

```diff
@@ -41,24 +41,55 @@
     end
   end
 
-  context 'when new stock is not importable' do
-    let!(:current_stock) { create :stock, :of_july, :completed }
-    let(:remote_stock)   { build :stock, :of_july, :pending }
+  context 'when remete stock is not importable' do
+    context 'because current stock completed' do
```

Comment:
Je trouve pas la tournure très clair, plutôt un truc comme "because it's already imported". Pareil pour le message dans les logs, en le lisant je comprends pas pourquoi il est pas importable. Un log genre `"Last stock available (from month #{remote.month}) already imported."` ça marche mieux.

Reply:
ok pour les propositions, à la différence que je vais ajouter 'completed' car ça correspond au status du stock plutôt que imported.
"because it's already imported (COMPELTED)"

Reply:
Ouais parce que le soucis c'est que quand tu lis les logs tu veux pas avoir à ouvrir les specs pour trouver le commentaire qui te fourni la table de vérité pour comprendre ce qu'il se passe ^^

Perso j'aurais fait une opération, mais tant que les logs sont clairs ça me va.

Reply:
mis à jour